aws/modules_common.tf

```diff
@@ -29,10 +28,24 @@ module "codebuild_ci_cid-checker" {
   ]
 }
 
-module "codebuild_cd_cid-checker" {
-  count                    = local.is_mainnet_envs
+module "codebuild_cd_cid-checker_watcher_calibrationnet" {
```

Comment:
I know it is more about naming. but don't need to put _calibrationnet in the module name without any strong reason.
It can be a reason for misunderstanding logic and where it has to be deployed.
Moreover, we have the count attribute that explicitly shows where it will be deployed